heartbeat/anything

```diff
@@ -206,7 +213,12 @@ user="$OCF_RESKEY_user"
 [ -z "$user" ] && user=root
 
 anything_validate() {
-	if ! su - $user -c "test -x $binfile"
+	if [ "$use_environment" = "yes" ]; then
```

Comment:
You should move this in front of the `case` line for all the actions to avoid having duplicate code in the functions.

FYI. You can do `git commit --amend` to update your commit and `git push -f` to force-push to update the commit in your PR.